packages/lisk-core/src/cli/actions/start.js

```diff
@@ -0,0 +1,11 @@
+const config = require('../../helpers/config');
+const Controller = require('../../controller');
+const ComponentFactory = require('../../factories/components');
+
+const logger = ComponentFactory.create('logger', config.components.logger);
```

Comment:
We can create helpers/logger.js:

```js
const liskBunyan = require('@lisk/lisk-logger-bunyan');
const config = require('../helpers/config');

 const logger = liskBunyan(config.components.logger);

 module.exports = logger;
```

This will reduce the number of imports we have to make to use logger instance and we don't need ComponentFactory class.